packages/attendance/src/components/AttendanceComponent.js

```diff
@@ -162,7 +162,11 @@ export const MultipalAttendance = ({
   const groupExists = (classObject) => classObject?.id;
   const markAllAttendance = async () => {
     setLoading(true);
-    if (typeof students === "object" && students.length > 0) {
+    if (
+      typeof students === "object" &&
+      students.length > 0 &&
+      moment().format("HH:MM") <= manifest?.["class_attendance.submit_by"]
```

Comment:
isn't this compared with the config from backend value stored using admin-console config ui?

Reply:
Yes